tedana/gscontrol.py

```diff
@@ -137,77 +137,67 @@ def gscontrol_mmix(optcom_ts, mmix, mask, comptable, ref_img, out_dir='.'):
     Filename                  Content
     ======================    =================================================
     sphis_hik.nii             T1-like effect
-    hik_ts_OC_T1c.nii         T1-corrected BOLD (high-Kappa) time series
-    dn_ts_OC_T1c.nii          Denoised version of T1-corrected time series
-    betas_hik_OC_T1c.nii      T1 global signal-corrected components
-    meica_mix_T1c.1D          T1 global signal-corrected mixing matrix
+    hik_ts_OC_MIR.nii         T1-corrected BOLD (high-Kappa) time series
+    dn_ts_OC_MIR.nii          Denoised version of T1-corrected time series
+    betas_hik_OC_MIR.nii      T1 global signal-corrected components
+    meica_mix_MIR.1D          T1 global signal-corrected mixing matrix
     ======================    =================================================
     """
-    LGR.info('Performing T1c global signal regression to remove spatially '
-             'diffuse noise')
-    RepLGR.info("T1c global signal regression was then applied to the "
-                "data in order to remove spatially diffuse noise.")
+    LGR.info("Performing minimum image regression to remove spatially-diffuse noise")
+    RepLGR.info(
+        "Minimum image regression was then applied to the "
+        "data in order to remove spatially diffuse noise."
+    )
 
     all_comps = comptable.index.values
-    acc = comptable[comptable.classification == 'accepted'].index.values
-    ign = comptable[comptable.classification == 'ignored'].index.values
+    acc = comptable[comptable.classification == "accepted"].index.values
+    ign = comptable[comptable.classification == "ignored"].index.values
     not_ign = sorted(np.setdiff1d(all_comps, ign))
 
     optcom_masked = optcom_ts[mask, :]
     optcom_mu = optcom_masked.mean(axis=-1)[:, np.newaxis]
```

Comment:
If we're naming everything based on stat names, shouldn'lt we make this `optcom_mean` instead of `optcom_mu`? I figure since we have other style changes in this PR, this would be a good one to change this for.